tests/scripts/test_tools_image_update_digest.py

```
@@ -0,0 +1,235 @@
+
+def _run(script: Path, *args: str) -> subprocess.CompletedProcess[str]:
+    """Invoke the script with given args, capturing output.
+
+    Args:
+        script: Path to the copied script.
+        *args: CLI arguments to forward.
+
+    Returns:
+        Completed process with captured stdout/stderr.
+    """
+    return subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)